components/channel/cmd/main.go

```diff
@@ -64,6 +56,25 @@ func main() {
 		logger.Fatal("Invalid / Missing Environment Variables - Terminating")
 	}
 
+	ctx := context.Background()
+	port, _ := strconv.Atoi(httpPort)
+
```

Comment:
Double checking it's ok to ignore the error here?  Is the error handling below sufficient?

Reply:
Good catch, better to check it to be safe.